astropy/units/format/ogip.py

```diff
@@ -19,6 +19,7 @@
 import keyword
 import math
 import os
+import copy
```

Comment:
Yikes, not good that that was missing. Which obviously means that the corresponding code never got run... But maybe best addressed in a coverage review at some point.